src/shared/consts.tsx

```diff
@@ -324,7 +324,19 @@ export const reportHeader = [
   'activity_end_time',
   'flag',
   'secret_user_id',
-  'userId',
+  'user_id',
```

Comment:
This should be `userId` as it will be a breaking change for scripts running on data exports.

Reply:
Updating

Reply:
These fields are accounted for in the current communication about changes to the report file.